src/app/App.tsx

```diff
@@ -83,6 +85,15 @@ export const App = (): JSX.Element => {
         </Notification>
       </Section>
     );
+  } else if (configErrors) {
```

Comment:
This is dangerous in a sense that if any generic configuration error comes through a user will be unable to use MAAS UI. We need to narrow this down to Vault errors or display a simple notification instead of replacing entire content of the app.